bin/cron/update_dotd

```diff
@@ -25,6 +25,9 @@ def main
   # Set the new developers room topic
   new_topic = current_topic.sub(/^.+?;/, "DOTD: @#{primary_dotd_user};")
   Slack.update_topic('developers', new_topic) unless new_topic == current_topic
+
+  # Remind dotd to update the flaky test tracker at 8am PST.
+  Slack.remind(Slack.user_id(DevelopersTopic.dotd), Time.now.to_i + 3600, 'Please update the <https://docs.google.com/spreadsheets/d/1DD6ebHM-2fsiMxxYCzhJYd1DzA69Z20qrCItaER9vto|Flaky Test Tracker>')
```

Comment:
maybe update_dotd was not an ideal suggestion, because it gets run twice, to avoid needing to be updated twice a year for timezone changes: https://github.com/code-dot-org/code-dot-org/blob/b6bde7529a20316a396cfe871c1634d94e38e268/cookbooks/cdo-apps/templates/default/crontab.erb#L64

as a result, I would expect this reminder to get posted twice.

my recommendation would be to expand the `unless new_topic == current_topic` condition above and include this reminder within that block, as a hacky way of making this only get run once

Reply:
my other comment here, perhaps more important, is that this message seems a bit un-actionable. What are you expecting the dotd to do in response to this message? I am wondering if the message on each DTT failure is perhaps the most timely and effective way to prompt the dotd.

Reply:
@davidsbailey I changed the mechanics to tag the DOTD and print out the message when a test build fails. I also added a bit more context to the message.

I know this will tag the DOTD twice on a build failure, but I think that will be helpful temporarily to make sure that developers see the new message.